test/unit/index-lambda-cat-bot.test.js

```diff
@@ -1,4 +1,69 @@
-test('this is a placeholder test', () => {
-  console.log('hello, i am running this test!!');
-  expect(true).toBe(true);
+const { handler } = require('../../src/index-lambda-cat-bot');
+const index = require('../../src/index-lambda-cat-bot');
+const s3 = require('../../src/services/s3');
+const twitter = require('../../src/services/twitter');
+const fakeImages = require('../data/s3-response-list-objects-v2.json').Contents;
+const uploadImageAPIResponse = require('../data/twitter-response-upload-image.json');
+
+jest.mock('s3');
+jest.mock('twitter');
+
+afterEach(() => {
+  jest.resetAllMocks();
 });
+
+test('handler exists', () => {
```

Comment:
I think you can safely remove this `expect(handler()).toBeDefined();` test since our other tests will implicitly test this anyway 🪄

Reply:
lol i forgot to delete this